make_dataset.py

```diff
@@ -13,6 +13,7 @@
 
 from mdtk import degradations, downloaders, data_structures, midi
 from mdtk.filesystem_utils import make_directory, copy_file
+from mdtk.pytorch_datasets import df_to_command_str
```

Comment:
This import requires pytorch to be installed. See #54

Reply:
You mention this in a TODO as well.

Reply:
yeah, I think we agreed it was ok to have this as a dependency? If not, we should move those functions out of pytorch_datasets into another module (i'd prefer this anyway)

Reply:
Yeah, the dependency is ok, but in case people can't, and are trying to run this script without pytorch somehow I mean. Moving it into another module is perfect.